.github/workflows/develop.yml

```diff
@@ -7,15 +8,15 @@ on:
       - develop
 
 jobs:
-  commitlint-lint-test-build:
+  lint-test-build:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Commit lint
-        uses: wagoid/commitlint-github-action@v4
+      # - name: Commit lint
+      #   uses: wagoid/commitlint-github-action@v4
```

Comment:
Why not just delete it? Isn't that what git history is for?

Reply:
done